Makefile

```diff
@@ -48,6 +48,7 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 .PHONY: fmt
 fmt: ## Run gofmt against code.
 	gofmt -s -w $(shell find . -name '*go' | grep -v mock)
+	gci write --skip-generated -s standard,default -s 'prefix(github.com/aws/eks-anywhere-packages)' $(shell find . -name '*go' | grep -v mock)
```

Comment:
I promise I won't ask for too many things to be broken out into separate PRs, but this one I have to ask about. The reason being that this change is probably responsible for a bunch of small styling changes in this PR, and this PR is already large enough without me having to try and figure out if the changes are just styling.

Also, I wouldn't mind if this change were discussed as a quick post-standup discussion in case anyone has any feelings about the tools we're using. If possible, I'd love if we could pick a single one of the 18 billion go formatting tools out there, and just stick with it. :) I'm exaggerating a little, but honestly, I can't think of a single thing that one formatter has over another that actually matters to me. Others may feel differently of course.

Reply:
That was to be compliant with github actions

Reply:
I'm throwing this change to https://github.com/aws/eks-anywhere-packages/pull/326 to investigate alternatives later